tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc

```diff
@@ -367,6 +406,44 @@ TEST(LinalgBackendEigen, SGMatrix_cholesky_solver)
 	EXPECT_EQ(x_ref.size(), x_cal.size());
 }
 
+TEST(LinalgBackendEigen, SGMatrix_ldlt_solver)
+{
+	const index_t size = 3;
+	SGMatrix<float64_t> A(size, size);
+	A(0, 0) = 0.0;
+	A(0, 1) = 0.0;
+	A(0, 2) = 0.0;
+	A(1, 0) = 0.0;
+	A(1, 1) = 1.0;
+	A(1, 2) = 2.0;
+	A(2, 0) = 0.0;
+	A(2, 1) = 2.0;
+	A(2, 2) = 3.0;
+
+	SGVector<float64_t> b(size);
+	b[0] = 0.0;
+	b[1] = 5.0;
+	b[2] = 11.0;
+
+	SGVector<float64_t> x_ref(size), x(size);
+	x_ref[0] = 0.0;
```

Comment:
I would prefer a test that is more self explaining.

Like you do
x = A*b

then you solve b2= A^-1 * x
and then you check that b==b2

But this one is OK as well